### PR TITLE
Reduce some empty line && code, add some TODO comments, fix typo

### PR DESCRIPTION
**Pull request by shiywang**
@smarterclayton PTAL, and if you don't mind, I would like to take care of those `move me to kcmdutil` code when upstream merged, Is that fine ?  
Thanks

### CODE REVIEW

**Comment by pweil-** [comment]
@openshift/cli-review

**Comment by smarterclayton** [comment]
The upstream Pr for that *should* have merged, but will have to double check.

**Comment by shiywang** [comment]
yeah, I mean after cherry pick, merge into our code.

File: pkg/cmd/cli/cmd/extract.go [hunk 1]
```diff
@@ -69,6 +69,7 @@ func NewCmdExtract(fullName string, f *clientcmd.Factory, in io.Reader, out, err
 		Run: func(cmd *cobra.Command, args []string) {
 			kcmdutil.CheckErr(options.Complete(f, in, out, cmd, args))
 			kcmdutil.CheckErr(options.Validate())
+			// TODO: move met to kcmdutil
```
Comment (fabianofranz) [inline pkg/cmd/cli/cmd/extract.go]: "move me"?

Comment (shiywang) [inline pkg/cmd/cli/cmd/extract.go]: yeah, I was meant to type "move me", but there are lots of "move met" exist, I thought it was like kind of cipher or something, anyway, will update all this typo :smile:

**Comment by shiywang** [comment]
@fabianofranz updated, ptal

**Comment by fabianofranz** [comment]
@shiywang thanks!
[merge]

**Pull request merged**